services/catalog/service/migrations/committed/000011-channel-localizations-added.sql

```diff
@@ -0,0 +1,31 @@
+--! Previous: sha1:c9457b917399b9a43ac0fc04d0e130cf728bfcb7
+--! Hash: sha1:7afd39916d603525b3b67b5a836d90b445e8bf2b
+--! Message: channel-localizations-added
+
+CREATE TABLE IF NOT EXISTS app_public.channel_localizations(
+  id INTEGER PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+  channel_id TEXT REFERENCES channel ON DELETE CASCADE,
+  locale TEXT NOT NULL,
+  is_default_locale BOOLEAN NOT NULL,
+  title TEXT,
```

Comment:
The title should be a required property or? We should then mark the field as "not null".